src/main.rs

```diff
@@ -54,23 +51,23 @@ struct Args {
     #[clap(long, display_order = 4, default_value_t = 55555)]
     port: u64,
 
-    /// Number of iterations to run (skip) before measuring the latency.
+    /// Seconds to run (skip) before measuring the latency.
     #[clap(
         long,
         display_order = 5,
         value_name = "ITERATIONS",
```

Comment:
Why is this still called ITERATIONS if everything is switched to Seconds? Overlooked?

Reply:
Overlooked and fixed